src/app/credential-issuer/middleware.js

```diff
@@ -1,13 +1,8 @@
-const {
-  API_BASE_URL,
-  API_CRI_CALLBACK,
-  EXTERNAL_WEBSITE_HOST,
-} = require("../../lib/config");
-const { generateJsonAxiosConfig } = require("../shared/axiosHelper");
+const { API_CRI_CALLBACK, EXTERNAL_WEBSITE_HOST } = require("../../lib/config");
 const { handleBackendResponse } = require("../ipv/middleware");
 const { logCoreBackCall, transformError } = require("../shared/loggerHelper");
 const { LOG_COMMUNICATION_TYPE_REQUEST } = require("../shared/loggerConstants");
-const axios = require("axios");
+const CoreBackService = require("../../services/coreBackService");
```

Comment:
I think I'd normally camelCase this (i.e. coreBackService), or even just destructure the imports directly?

Hopefully we'll move to TS and ES6 modules soon anyway...

Reply:
I was getting struggling to avoid errors when destructing. I'll change it to camelCase